### ISSUE TITLE
Cant upload android apk via appcenter_upload with 1.9.0

### ISSUE THREAD

**Issue by filipkowicz** [microsoft/fastlane-plugin-appcenter#246]
from what i see backend respond with 200 and empty body for [source](https://github.com/microsoft/fastlane-plugin-appcenter/blob/bf0b56f5d4d0d24017f600976b8c3335dd2da84f/lib/fastlane/plugin/appcenter/helper/appcenter_helper.rb#L251) call

action is failing with "Error uploading binary" and "Upload aborted"
for 1.8.1 with same params works well

**Comment by filipkowicz** [comment]
side note - add 
` ENV['DEBUG'] = "true"`
before calling action

**Comment by joseignacioojeda** [comment]
I am having the same problem

**Comment by filipkowicz** [comment]
I've give up and switch to 1.8.1

**Comment by lampietti** [comment]
same here, and it was working fine yesterday before the last incident https://status.appcenter.ms/ ...

**Comment by leo-pedidosya** [comment]
Same here

**Comment by PatrykKaczmarek** [comment]
Same here. Interestingly building (and uploading binary) the same project with exactly the same configuration passes when using bitrise CLI locally instead of building it on their servers.

**Comment by sedwo** [comment]
Been struggling with this all evening.  I concur, `fastlane-plugin-appcenter (1.9.0)` breaks the upload process.

![image](https://user-images.githubusercontent.com/1434449/94628033-03a94000-028d-11eb-8346-f651ac0c364c.png)

**Comment by NirGrocerKey** [comment]
having the same issues since the last incident, any help how to roll back to 1.8.1?

**Comment by b3hni4** [comment]
I have the same issue, workaround is to use fastlane 2.160.0 until this is fixed

**Comment by alexv-anyvision** [comment]
Having same issue

**Comment by neil-gebbie-smarterley** [comment]
This is a duplicate of https://github.com/microsoft/fastlane-plugin-appcenter/issues/247 - for the time being you can lock your version to 1.8.1 by following these instructions - https://docs.fastlane.tools/plugins/using-plugins/#plugin-source

`gem 'fastlane-plugin-appcenter', "1.8.1"`

**Comment by ff-ahumberston** [comment]
Having attempted the version local to 1.8.1, we are now seeing the following error:

`INFO [2020-09-30 11:34:12.10]: ------------------------------
INFO [2020-09-30 11:34:12.10]: --- Step: appcenter_upload ---
INFO [2020-09-30 11:34:12.10]: ------------------------------

Looking for related GitHub issues on fastlane/fastlane...
Search query: undefined method `length' for nil
`

Is anybody else experiencing this or have a suggested resolution.

**Comment by fobo66** [comment]
Here's what I've caught in logs:

```
fastlane finished with errors
 [!] Client error: 400: {"error"=>{"code"=>400, "message"=>"The request contains invalid content."}}
```

**Comment by baxri** [comment]
Hey all have same problem 

```
fastlane finished with errors
 [!] Client error: 400: {"error"=>{"code"=>400, "message"=>"The request contains invalid content."}}
```

**Comment by ff-ahumberston** [comment]
According to AppCenter support, dropping back to fastlane 2.160.0 will resolve the issue in the interim.

I'm currently struggling to drop back to that version, even though I have it defined in the ~/Gemfile

**Comment by dileep017** [comment]
reverting to 2.160.0 also has not resolved the issue, and using the latest one is giving 
fastlane finished with errors
 [!] Client error: 400: {"error"=>{"code"=>400, "message"=>"The request contains invalid content."}}

**Comment by baxri** [comment]
> reverting to 2.160.0 also has not resolved the issue, and using the latest one is giving
> fastlane finished with errors
> [!] Client error: 400: {"error"=>{"code"=>400, "message"=>"The request contains invalid content."}}

Same for me on both versions latest and previsus versions

**Comment by filipkowicz** [comment]
@baxri @dileep017 @ff-ahumberston please note that we are proposing reverting version of plugin not fastlane itself from 1.9.0 to 1.8.1 -> see 

> This is a duplicate of #247 - for the time being you can lock your version to 1.8.1 by following these instructions - [docs.fastlane.tools/plugins/using-plugins/#plugin-source](https://docs.fastlane.tools/plugins/using-plugins/#plugin-source)
> 
> `gem 'fastlane-plugin-appcenter', "1.8.1"`

**Comment by neil-gebbie-smarterley** [comment]
After my solution above (https://github.com/microsoft/fastlane-plugin-appcenter/issues/246#issuecomment-701246375) I think you might also need to generate a new Gemfile.lock

**Comment by Karthi27p** [comment]
> According to AppCenter support, dropping back to fastlane 2.160.0 will resolve the issue in the interim.
> 
> I'm currently struggling to drop back to that version, even though I have it defined in the ~/Gemfile
> 
> Edit - reverting to 2.160.0 fails to resolve the issue for ourselves.

Reverting to fastlane version 2.160.0 works. Try using the below commands to reinstall fastlane of specific version

gem uninstall fastlane
gem install fastlane --version 2.160.0

**Comment by ff-ahumberston** [comment]
Reverting to fastlane 2.160.0 gives the following error when attempting to upload 
`NoMethodError: [!] undefined method length for nil:NilClass`

Dropping back the version of the fastlane-plugin-appcenter to 1.8.1 gives similar.

**Comment by ifs-hichemlaroussi** [comment]
On my case, only this combination works fine : 
[Fastline] `fastlane_version "2.161.0"`
[Pluginfile] `gem 'fastlane-plugin-appcenter', "1.8.1"`

**Comment by andrewdmontgomery** [comment]
Reverting the AppCenter plugin to `1.8.1` (from `1.9.0`) is working for us so far.

**Comment by vanlooverenkoen** [comment]
How do I revert back? 

this is added to my PluginFile but 1.9.0 is the used version
```
gem 'fastlane-plugin-appcenter', "1.8.1"
```

**Comment by andrewdmontgomery** [comment]
Something similar was happening in the scripting we used to run fastlane.  You'll likely want to make sure you are running fastlane via `bundle exec`:

```
bundle exec fastlane :lane_name
```

Note that you will need to run `bundle install` (or some version of that) before you can run fastlane with bundler.  The `bundle install` command will use the Gemfile, the Gemfile.lock (once it exists), and the `fastlane/Pluginfile` (which is called from within the Gemfile) to make sure you have the fastlane gem, the Fastlane-plugin-appcenter gem, and all of the necessary dependencies – all with the correct versions.

Then prefixing your fastlane command with `bundle exec` will make sure that you are using that whole setup, rather than some other installation of fastlane that isn't using your Gemfile and Pluginfile to pin versions.

See this page for more info: https://docs.fastlane.tools/getting-started/ios/setup/

**Comment by vanlooverenkoen** [comment]
I used homebrew for my fastlane installation. For now I will use bundle exec untill it is fixed

**Comment by lumaxis** [comment]
This is likely a duplicate of #247 and should be fixed with `v1.10.0` 🙏🏼
